packages/@o3r/dev-tools/src/cli/aql-artifact-cleaner.ts

```diff
@@ -64,14 +64,15 @@ if (!opts.basicAuth && !opts.apiKey) {
   process.exit(1);
 }
 let authHeader: Headers;
```

Comment:
```ts
const authHeader: Headers = {
  ...(
   // eslint-disable-next-line @typescript-eslint/naming-convention -- naming convention required by HTTP
   opts.basicAuth ? {Authorization: 'Basic ' + (opts.basicAuth as string)} : {'X-JFrog-Art-Api': (opts.apiKey as string)}
  )
};
```
maybe a question of preference 
but I think ternary are easier to read 
plus in this case we can change `authHeader` into a `const` so it brings more information